File: lesson5/python3/lesson5_1.py
#!/usr/bin/python3
#: lesson5/python3/lesson5_1.py
# просто процедура

def error():
    print("ОШИБКА! Число меньше чем ноль!")


def correct_answer():
    print("ПРАВИЛЬНО! Число больше чем ноль!")
    

value_numb = int(input("Введите натуральное число больше чем ноль => "))

if value_numb < 0:
    error()
else:
    correct_answer()

File: lesson5/c_c++/lesson5_1.c
//: lesson5/c_c++/lesson5_1.c
#include <stdio.h>

void Error() {
    printf("ОШИБКА! Число должно быть больше чем ноль.\n");
}

void correct_answer() {
    printf("ПРАВИЛЬНО! Ответ больше чем ноль.\n");
}

int main() {
    int value_numb;
    printf("Введите натуральное число больше чем ноль => ");
    scanf("%d", &value_numb);

    if (value_numb < 0) {
        Error();
    } else {
        correct_answer();
    }
    return 0;
}
